lib/stdlib/src/erl_tar.erl

```diff
@@ -1019,11 +1019,14 @@ do_get_format({error, _} = Err, _Bin) ->
 do_get_format(#header_v7{}=V7, Bin)
   when is_binary(Bin), byte_size(Bin) =:= ?BLOCK_SIZE ->
     Checksum = parse_octal(V7#header_v7.checksum),
-    Chk1 = compute_checksum(Bin),
-    Chk2 = compute_signed_checksum(Bin),
-    if Checksum =/= Chk1 andalso Checksum =/= Chk2 ->
+    BadChecksum = case compute_checksum(Bin) of
```

Comment:
I'd probably rename `BadChecksum` to `IsBadChecksum` just to infer that it's a boolean rather than a Checksum Binary

Reply:
Done.